editor/dependency_editor.cpp

```diff
@@ -349,7 +349,7 @@ void DependencyEditorOwners::show(const String &p_path) {
 	_fill_owners(EditorFileSystem::get_singleton()->get_filesystem());
 	popup_centered_ratio(0.3);
 
-	set_title(TTR("Owners Of:") + " " + p_path.get_file());
+	set_title(TTR("Owners of:") + " " + p_path.get_file() + " " + vformat(TTR("(Total: %d)"), owners->get_item_count()));
```

Comment:
Can probably use a single format call like `vformat(TTR("Owners of: %s (Total: %d)"), ...)`. So languages with different punctuation & space rules can benefit from it.

e.g. CJK fullwidth punctuations don't need a space after `：` and before `（`.